lib/playground.dart

```diff
@@ -669,12 +668,12 @@ class Playground implements GistContainer, GistController {
 
   /// Analyzes and runs the gist.  Auto-runs the gist if [autoRun] is true and
   /// the analyzer comes back clean.
-  Future<void> _analyzeAndRun({bool autoRun = false}) {
+  Future<void> _analyzeAndRun({bool autoRun = true}) {
```

Comment:
I believe L619 is the only place this gets called, so you could just take the parameter out.